scom/stackstate_checks/scom/scom.py

```diff
@@ -82,114 +52,136 @@ def sendAlerts(self, session, scom_ip, domain, username, password, component_id)
                     "time_added:%s" % event.get("timeadded")
                 ]
             })
+    def get_health_state(self,healthIconUrl):
+        if re.search('StatusOkComplete', healthIconUrl, flags=re.IGNORECASE) or re.search('healthy_ex', healthIconUrl,flags=re.IGNORECASE):
+           return 'Healthy'
+        elif re.search('StatusWarning', healthIconUrl, flags=re.IGNORECASE) or re.search('warning_ex', healthIconUrl,flags=re.IGNORECASE):
+           return 'Deviating'
+        elif re.search('StatusError', healthIconUrl, flags=re.IGNORECASE) or re.search('critical_ex', healthIconUrl,flags=re.IGNORECASE):
+           return 'Error'
+        elif re.search('StatusUnknownGreen', healthIconUrl, flags=re.IGNORECASE) or re.search('maintenance_ex',healthIconUrl,flags=re.IGNORECASE):
+           return 'Not Monitored'
+        else:
+           return str(healthIconUrl.split("/")[-1])
+
+    def get_type(self,session,component_id,domain, username, password, scom_ip):
+        if self.requests_counter > self.requests_threshold:
```

Comment:
Same here, let's log it

Reply:
i log it at the end because if i add it after every threshold check, then it's going to be logged multiple times
self.log.info("maximum number of requests reached! "+str(self.requests_counter))
should i add it after each threshold check ?